src/components/scenes/ManageTokensScene.tsx

```diff
@@ -21,19 +22,32 @@ import { FilledTextInput } from '../themed/FilledTextInput'
 import { ManageTokensRow } from '../themed/ManageTokensRow'
 import { SceneHeader } from '../themed/SceneHeader'
 import { Title } from '../themed/Title'
+import { WalletListSectionHeader } from '../themed/WalletListSectionHeader'
 import { ButtonsViewUi4 } from '../ui4/ButtonsViewUi4'
 import { CryptoIconUi4 } from '../ui4/CryptoIconUi4'
 
+/**
+ * walletId: ID of the wallet whose tokens we are managing
+ * newTokenIds: We routed from a notification that was triggered from detecting
+ * these new tokenIds having nonzero balance. Show these at the top of the list.
+ */
```

Comment:
I would annotate the members themselves, so you get the tooltip when you hover the exact item:

```ts
export interface ManageTokensParams {
  /** The wallet whose tokens we are managing */
  walletId: string

  /**
   * Show these at the top of the list, since we routed from
   * the new-token notification pop-up.
   */
  newTokenIds?: string[]
}
```